### DISCUSSION TITLE
HTML Code in Button and devicename after 13.3.0.5 not possible

### DISCUSSION THREAD

**Discussion by karoCB** [arendst/Tasmota#21826] [General]
`Button1 aktion<p>max%VAR1%_min%VAR2% or 
Devicename <h6> Information_%VAR3%`

From release 13.3.0.5 to 14.1 !  Is the refusal to use HTML.  
Is there a new solution for Rule and Script or is this a bug?

**Comment by sfromis** [comment]
The old unsafe code you relied on about not sanitizing strings was fixed quite some time ago.

**Comment by karoCB** [comment]
the old panel is then a problem 
![IMG_7856](https://github.com/user-attachments/assets/e4d50ab9-d17a-43dd-b63e-f3d09918b80a)
